README.md

```diff
@@ -70,6 +70,11 @@ The read_pattern_multi function is design to read in multiple pattern files and
 
 The merge_core_pattern function is designed to take a patterns DF and cross examine it with a core DF. The resulting pandas DF will be a DF with all of the values from your patterns DF as well as the matching values from your core DF (merge done on 'safegraph_place_id'). All Pandas arguments and keywork arguments can be passed into this function.
 
+#### merge_socialDist_by_dates(path_to_social_dist, start_date, end_date, args, kwargs)
+
+The merge_socialDist_by_dates function is designed to merge the social distancing data from a given start_date date to a given end_date. The resulting pandas DF will be a DF of all social distancing data from the start_date to the end_date. All Pandas arguments and keywork arguments can be passed into this function.
```

Comment:
in the first sentence, it says "from a given start_date date"

in the last sentence the pandas argument sentence has a typo - 'keywork' instead of 'keyword' (that was my fault, I just fixed all of the instances in the original with that same type

Reply:
Thanks for pointing that out, must have over read it after updating some of the names to help the clarity